etcd-io/etcd/etcd.go

```diff
@@ -54,7 +54,7 @@ func New(l log.Logger, kubectl *kubectl.Kubectl, opts ...Option) (*ETCD, error)
 	if err := viper.UnmarshalKey(inst.configKey, &inst.cfg); err != nil {
 		return nil, err
 	}
-	if err := os.MkdirAll(inst.cfg.ConfigPath, 0o700); err != nil {
+	if err := files.MkdirAll(inst.cfg.ConfigPath); err != nil {
```

Comment:
Why not just use the OS variant? What's the benefit of wrapping the whole thing?

Reply:
1. one place for 0700 definition
2. one place for added "" check